lib/index.js

```diff
@@ -26,19 +31,19 @@ function preprocessor(css, options) {
 
   options = options || {};
 
-  var browserConfig = options.browsers || [
-    'Explorer >= 9',
-    'last 2 Chrome versions',
-    'last 2 Firefox versions',
-    'last 2 Safari versions',
-    'last 2 iOS versions',
-    'Android 4'
-  ];
-
-  css = rework(css, options).use(suit(options)).toString(options);
-
-  // vendor prefixes
-  css = autoprefixer(browserConfig).process(css).css;
+  css = postcss([
+    atImport({
+      root: options.root,
+      transform: function(css, filename) {
+        return postcss([bemLinter, reporter]).process(css, {from: filename}).css;
+      }
+    }),
+    customProperties,
+    calc,
+    customMedia,
+    autoprefixer({browsers: options.browsers}),
```

Comment:
Yeah I was wondering about this. I think what was there before was going back two versions which is what `autoprefixer` does now anyway (I think that was written when it was on v1 or something) but we could add a default list to ensure `IE10` gets included now that we have flexbox kicking around.